styles/mark-the-words.css

```diff
@@ -45,6 +46,8 @@
 /* Colors and styling for word selections */
 .h5p-mark-the-words [aria-selected="true"] {
   background: #cee0f4;
+  border: 1px solid #1356a3;
```

Comment:
@Gwenillia : I believe it is considered best practice to only add CSS that changes the behavior. I.e: border is already 1px solid, the only change is the color. The only change needed here is therefore `border-color:  #1356a3`. The same goes for the other overrides below.